packages/blockchain-link/src/workers/baseWebsocket.ts

```diff
@@ -116,6 +118,8 @@ export abstract class BaseWebsocket<T extends EventMap> extends TypedEmitter<T &
     }
 
     protected sendMessage(message: Record<string, any>) {
+        this.options.onSend?.(message);
```

Comment:
imho it should be called few lines below if there was no error.

from what i understand this is used only for debug purposes right? some comment/note would be nice

at first sight i was thinking (basing by the name) that this is a callback after successful send (B-L in telling you that he did send something)

Reply:
Renamed to `onSending` and moved.
https://github.com/trezor/trezor-suite/pull/9101/commits/fcee225fbab98997736c83a77c67aba4c7a01c9c https://github.com/trezor/trezor-suite/pull/9101/commits/84df50bb4a83952391844b01bcff612bc71d137b